validators/cgroup_validator_linux.go

```diff
@@ -44,10 +47,31 @@ func (c *CgroupsValidator) Name() string {
 }
 
 const (
-	cgroupsConfigPrefix = "CGROUPS_"
-	unifiedMountpoint   = "/sys/fs/cgroup"
+	cgroupsConfigPrefix      = "CGROUPS_"
+	defaultUnifiedMountpoint = "/sys/fs/cgroup"
+	mountsFilePath           = "/proc/mounts"
 )
 
+// getUnifiedMountpoint will check /proc/mounts and return the cgroup dir
+func getUnifiedMountpoint() string {
+	f, err := os.Open(mountsFilePath)
+	if err != nil {
```

Comment:
fstype should be checked to confirm whether defaultUnifiedMountpoint is actually cgroupfs.

Reply:
if we cannot open the `/proc/mounts`(err!=nil here), how can we confirm about the default mountpoint groupfs?

- check `/proc/self/mountinfo`?

Reply:
https://github.com/kubernetes/system-validators/pull/41/commits/95629f0e852ddc8c74bdb25c98cf144509146018 tried to address this.

 1. check /proc/self/mountinfo if the default mount point is available
 2. check /proc/mounts and return the cgroup dir

Reply:
@neolit123 @KentaTada  PTAL